CompetitionTemplate/tests/competition/subsystems/arm/ArmSubsystemTest.java

```diff
@@ -68,6 +68,12 @@ public void limitSwitchTest() {
     }
     
     @Test
+    public void maxAngleHeightTest(){
+        setMockEncoder(90);
+        assertTrue(armSubsystem.isArmAtMaxAngleHeight());
+    }
+    
+    @Test
     public void testCalibration() {
```

Comment:
There's also a test that was disabled here we can now enable and update with this bit:
```
    @Test
    @Ignore("Currently no upper limit switch on robot")
    public void limitSwitchTest() {
```
Remove the `@Ignore` and update the test to set the angle